From 525e5e604d1570ab8f72eb19117d52b909ee38a4 Mon Sep 17 00:00:00 2001
From: Lionel Debieve <lionel.debieve@foss.st.com>
Date: Fri, 3 Feb 2023 08:48:07 +0100
Subject: [PATCH 0067/1141] arm64: defconfig: add arm watchdog config in addons
 fragment

Add the CONFIG_ARM_SMC_WATCHDOG to enable the ARM watchdog support.

Signed-off-by: Lionel Debieve <lionel.debieve@foss.st.com>
Change-Id: I1be0f41cabc667968fc63bffa36d7bcead7aff0f
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/288993
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/configs/fragment-02-defconfig-addons.config | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/arm64/configs/fragment-02-defconfig-addons.config
+++ b/arch/arm64/configs/fragment-02-defconfig-addons.config
@@ -1,5 +1,6 @@
 #  ALPHABETICALLY SORTED: $LC_ALL=C sort fragment-02-defconfig-addons.config -o fragment-02-defconfig-addons.config
 CONFIG_ARCH_STM32=y
+CONFIG_ARM_SMC_WATCHDOG=y
 CONFIG_DYNAMIC_DEBUG=y
 CONFIG_SERIAL_STM32=y
 CONFIG_SERIAL_STM32_CONSOLE=y
